HISTORY

```diff
@@ -1 +0,0 @@
-ChangeLog
```

Comment:
Why are you deleting these symlinks?

Reply:
It's mentioned in the commit message. I'm assuming that they only existed to avoid warnings from the bootstrap script. Without "foreign" in the automake options, when you run the bootstrap, it will complain about the README, NEWS, and HISTORY being missing, because those are required for a "gnu" project (which is the default if you don't specify "foreign").

Reply:
Sorry, should have read your commit messages. I knew what foreign was good for, just don't see why people don't like to have these files around.

Reply:
The `ChangeLog` is still there, I've only removed the sylinks from `NEWS` and `HISTORY` to it.